### ISSUE TITLE
Render Tag:historic=rune_stone

### ISSUE THREAD

**Issue by xztraz** [gravitystorm/openstreetmap-carto#2634]
Hi. This is a request to get rune stones rendered on the map. 

Nordic countries are full of old rune stones that would be nice to map out.
![image](https://cloud.githubusercontent.com/assets/25690871/26069882/ce991cc0-39a2-11e7-82f5-07bc1a8d05a7.png)

Usually included in local maps as this symbol
![image](https://cloud.githubusercontent.com/assets/25690871/26069782/7773d110-39a2-11e7-9af1-d890ea04b528.png)

http://wiki.openstreetmap.org/wiki/Tag:historic=rune_stone

Best regards

**Comment by xztraz** [comment]
https://commons.wikimedia.org/wiki/File:Runic_letter_raido.svg

**Comment by pnorman** [comment]
With 280 uses. it's not in general enough use to render.

Even with wider usage I'd still be sceptical on its value on a general-purpose map, but there's no need to discuss that while there's insufficient usage.

**Comment by xztraz** [comment]
In nordic contries these are marked on every map. Countryside is usually formed after those stones since they are protected by law and can't just be moved. Build around them.

These stones are of great historical value. Since they don't render i think interest of mapping them is not encouraged.

There are a couple of thousand stones in nordic contries and on other places the vikings discovered during their mapping of the world.

And just closing without discission.. come on..

**Comment by xztraz** [comment]
Here is sniplet of a typical swedish map
![image](https://cloud.githubusercontent.com/assets/25690871/26078459/eab3c000-39bf-11e7-91f9-6f6f7b031b2d.png)

**Comment by xztraz** [comment]
https://en.wikipedia.org/wiki/Runestone

**Comment by imagico** [comment]
This is an international map which makes it somewhat problematic to render things that are specific to a fairly small area.  So no matter what it is unlikely we will ever add what you desire, namely a specific rendering of runestones as you know it from local maps.

That being said there is of course the possibility to render these features the same way as we render other more generic features - like `historic=memorial` or `historic=archaeological_site`.  But there are a number of things to clarify before i think this can be considered:

* it is not clear if `historic=rune_stone` is actually the dominating tag for this kind of feature - `historic=memorial` - which is a more widely used tag that is already rendered (see http://wiki.openstreetmap.org/wiki/Tag:historic%3Dmemorial) seems a fairly good fit and is probably used for runestones already.
* spatial distribution of `historic=rune_stone` indicates this is not exclusively used for actual runestones, probably because the documentation is not very clear for someone who does not know what a rune is.

Long story short: if you can show `historic=rune_stone` is the dominating tag for runestones and the documentation on the wiki is improved to ensure it is not used for other things we could consider rendering it like `historic=memorial`.  But with only 290 uses it might be better to just move to using `historic=memorial` + `memorial=rune_stone` + `historic:civilization=viking` or similar.

**Comment by xztraz** [comment]
Thats a better answer :)
Yes Runes are quite localized to mainly sweden but they have a strong influence here and are interesting histoy free for everyone to see.

Sometimes the same symbol is used to mark an important archelogical site (followed by a marked area) or runes carved in a bedrock (also sloppishly called rune stone in sweden).
memorial could do but the symbol is quite missleading for a rune stone.

historic=memorial +
memorial=rune_stone
would be better so it gets marked as something at least and if numbers go up we could maybe have the R symbol later on.

Quite boring to mapp stuff thats invisible :)

So that would be a batch job to redo the runestones to memorial + rune and revrite the wiki to suggest this way of marking runes.

**Comment by xztraz** [comment]
viking could be left out it's kind of redundant.

**Comment by dieterdreist** [comment]
2017-05-15 23:43 GMT+02:00 xztraz <notifications@github.com>:

> historic=memorial +
> memorial=rune_stone
> would be better so it gets marked as something at least and if numbers go
> up we could maybe have the R symbol later on.
>
>

+1, seems suitable tagging, besides that according to wikipedia, it should
be "runestone", not "rune_stone".

Still according to wikipedia, some might be better mapped as boundary
stones: https://wiki.openstreetmap.org/wiki/Tag:historic%3Dboundary_stone

Also have a look at the historic:civilization key

**Comment by xztraz** [comment]
since rune stones where only made by vikings it's kind of redundant ish. but doesn't hurt. 

runestone should be used if thats how you spell it in english.

ok so

historic=memorial
memorrial=runestone
historic:civilization=viking

Other optional stuff?
inscription=*
year=*
idsomething=*   (stones are categorized with names like U102)

What should be used to indicate if it's a "Hällristning" rune carved into bedrock

Boundary.. hm. taken from wikipedia..
"A vast majority, 94%, are raised in memory of men"
So mostly memorial pieces i would say.

**Comment by imagico** [comment]
I would suggest to move the tagging related discussion to the tagging mailing list or the wiki.

**Comment by xztraz** [comment]
yeah probably. so much work to get some stones visible.. :p

**Comment by SomeoneElseOSM** [comment]
> so much work to get some stones visible

(apologies for the offtopic, but) as with anyone interested in displaying low-volume tags, I'd suggest creating your own map.  See https://switch2osm.org/manually-building-a-tile-server-16-04-2-lts/ for setting up a server that serves tiles in the same way that OpenStreetMap.org does, and https://ircama.github.io/osm-carto-tutorials/ for a bunch of really useful tutorials to help with e.g. modifying map styles.

**Comment by lutzla** [comment]
Why historic = memorial? historic = run_stone is ok!
http://gk.historic.place/historische_objekte/index.html?zoom=13&lat=59.10616&lon=17.14621&detail=3&select=n3065991117&pid=KmHaSaHe

**Comment by dieterdreist** [comment]
2017-05-20 9:01 GMT+02:00 lutzla <notifications@github.com>:

> Why historic = memorial? historic = run_stone is ok!



as it was explained above, historic=runestone is fine from a tagging
aspect, but will not be rendered in osm-carto (according to the
maintainers), so subtagging memorials as runestones would preserve the
information and would be rendered in the general style, this is why people
interested in this topic might consider tagging like this.

**Comment by lutzla** [comment]
Sorry, ich kann es nur auf Deutsch erklären.
Weitergedacht würde diese Herangehensweise bedeuten, jemand hätte gern heritage=* in osm-carto.
Die Entwickler sagen "Nein", und ich mappe jedes denkmalgeschützte Haus als historic=memorial, 
oder historic=monument mit memorial=heritage oder monument=heritage, dann wird es auf der Karte angezeigt, und die Information bleibt erhalten.
Ich will nicht Glauben, das Empfehlenswert ist!

**Comment by imagico** [comment]
To clarify: i did not say we won't render `historic=rune_stone`, i just said i would be against rendering it in a specific way unrelated to existing rendering of `historic=memorial` and at the moment i would be against rendering it at all because:

* it is a feature specific to a small area with a specific tagging.
* it is rarely used and it is not clear (not to say unlikely) that it is currently the dominating tag for runestones.
* the documentation of `historic=rune_stone` is fairly bad and fails to explain to the mapper unfamiliar with runic inscriptions and distinguishing them from other inscriptions when to use this tag and when not to use it.
* related to that a tag which is defined through a very specific cultural differentiation that is not always easy to determine for a mapper who is no archaeological expert might not be the best choice as a primary tag - a problem evident in use of it.

Auf Deutsch: Wir wollen hier nicht das Tagging bestimmen, welches die Mapper verwenden, aber wir rendern auch nicht jedes Tag, das sich jemand ausdenkt nur weil das gemappte Objekt im Prinzip darstellungswürdig ist.

**Comment by dieterdreist** [comment]
Am 21. Mai 2017 um 13:01 schrieb lutzla <notifications@github.com>:

> Weitergedacht würde diese Herangehensweise bedeuten, jemand hätte gern
> heritage=* in osm-carto.
> Die Entwickler sagen "Nein", und ich mappe jedes denkmalgeschützte Haus
> als historic=memorial,
> oder historic=monument mit memorial=heritage oder monument=heritage, dann
> wird es auf der Karte angezeigt, und die Information bleibt erhalten.
>

he wrote: following this principle. if someone would like to have
heritage=* rendered in osm-carto but the maintainers would reject the idea
he might come to the conclusion to map heritage sites as historic=memorial
or monument to have them rendered.

IMHO this is not comparable, because historic=memorial or monument usually
do not apply to heritage sites (although there will be some small overlap),
i.e. this would be tagging for the renderer (and would be reverted) and the
information about the thing (heritage) would not remain in the db.
Runestones on the other hand, are (according to what I have learnt in this
thread) fitting the bill of historic=memorial (they have been set up in
memory of something/someone).
